Source/Data/HACompressedEntity.swift

```diff
@@ -0,0 +1,75 @@
+import Foundation
+
+public struct CompressedStatesUpdates: HADataDecodable {
```

Comment:
i cannot really super well remember why i made the `HA` prefix for everything (i think i've been bit by namespace clashes too much) but it would be nice to preserve it